### ISSUE TITLE
Increase Color Difference Between Selected and Deselected UI Elements

### ISSUE THREAD

**Issue by skybaks** [openplanet-nl/openplanet-issues#346]
This is most apparent in the default style of ImGui in openplanet when using multiple tabs:
![image](https://github.com/openplanet-nl/issues/assets/52106022/41a50620-86bf-4daf-9452-a9870983a6a4)
It is somewhat difficult to identify which tab is the selected one out of the many shown.

I know that I can install Overlay Style Manager and pick from one of the many styles which have already solved this problem but many, many Openplanet users do not.
Additionally, I know I could write code in my plugin to tinker with the tab style colors but I would prefer not to do that because it would certainly impact those who _were_ using a custom UI style.

I think this change would improve the Openplanet user experience out of the box in terms of better accessibility.

**Comment by codecat** [comment]
I agree, this could use some more contrast. I'll have to play around with this. In the meantime, if you have any color suggestions, feel free to share!
